ccmlib/containers/scylla_node.py

```diff
@@ -0,0 +1,10 @@
+from ccmlib.containers.docker_client import BaseDocker
```

Comment:
i would call the whole file with a different name to reflect this is basically a docker container and not a `scylla_node` from the CCM perspective